README.md

```diff
@@ -148,3 +148,20 @@ Hello Parsec from the Parsec CLI Tool!
 
 Finished!
 ```
+
+# Running More Extensive Tests
+
+The `hello-parsec` container can optionally be used to execute a full suite of functional tests in
+place of the default demo. This is achieved by supplying an override to the `docker run` command so
+that it executes the `parsec-cli-tests.sh` script, which is included in the container image.
+
+Running the container as follows will execute the tests:
+
+```
+docker run -v /run/parsec:/run/parsec hello-parsec ./parsec-cli-tests.sh
```

Comment:
I would recommend to add `--rm` parameter to the example

Reply:
Done, thanks.